cmd/engflow_auth/main.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 type appState struct {
-	// These vars are initialized by `build()` iff they are not pre-populated;
+	// These vars are initialized by `build()` if they are not pre-populated;
```

Comment:
I think this was meant to be an abbreviation rather than a misspelling. Let's expand it.

```suggestion
	// These vars are initialized by `build()` if and only if they are not pre-populated;
```